tests/unit_tests/rigid_pendulum_poc/test_math_utilities.cpp

```diff
@@ -0,0 +1,79 @@
+#include <gtest/gtest.h>
+
+#include "src/rigid_pendulum_poc/utilities.h"
+
+namespace openturbine::rigid_pendulum::tests {
+
+TEST(MathUtilitiesTest, CloseTo) {
+    ASSERT_TRUE(close_to(1., 1.));
+    ASSERT_TRUE(close_to(1., 1. + 1e-7));
+    ASSERT_TRUE(close_to(1., 1. - 1e-7));
+    ASSERT_FALSE(close_to(1., 1. + 1e-5));
+    ASSERT_FALSE(close_to(1., 1. - 1e-5));
+    ASSERT_TRUE(close_to(1e-7, 1e-7));
```

Comment:
Why do you use ASSERT instead of EXPECT?
I don't think it matters very much, but ASSERT will stop the program right there, while EXPECT will let it continue so you can see how many failures there are, instead of just the first failure.

Reply:
Thanks for the helpful tip, @alanw0 - will use `EXPECT` statements going forward where it does not invalidate the rest of the test. 

For the above set,  I changed the tests to one `ASSERT_TRUE` statement per test per @ddement's suggestion. Let's keep them unchanged.